osbs/build/build_request.py

```diff
@@ -377,17 +377,34 @@ def adjust_for_registry_api_versions(self):
     def adjust_for_triggers(self):
         """
         Remove trigger-related plugins if no triggers set
+        import_image plugin is only removed for non-custom base images
```

Comment:
That's a good point. As is, it'll only get turned off if there are no `triggers` in `spec` (meaning autorebuild is disabled in the system). We could leave it enabled in this case as well, but it provides no benefit.